src/main/java/frc/robot/RobotMap.java

```diff
@@ -21,6 +21,9 @@
 
         int CLIMBER_WINCH_SPARK_MAX_1 = 12;
         int CLIMBER_WINCH_SPARK_MAX_2 = 13;
+
+        int CLIMBER_PLACER_TALON_LEFT = -1;
```

Comment:
Add TODO of changing port (to this, line 26, lines 43-46).
You don't have to if you remember to change it, but if you forget I'll make fun of you for not listening to me